src/styles/main.scss

```diff
@@ -1,3 +1,58 @@
-body {
-  margin: 0;
+$countDay: 7;
+$sizeDay: 100px;
+$days:
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.calendar {
```

Comment:
should be 7 days in a row 
![Screenshot 2023-04-03 at 20 18 21](https://user-images.githubusercontent.com/117587463/229581823-8386be37-f72b-4ea3-97a4-c229b173d714.png)